BibBuddy/app/src/main/java/de/bibbuddy/BibTexKeys.java

```diff
@@ -26,5 +26,6 @@
   public static final String OPENING_CURLY_BRACKET = "{";
   public static final String CLOSING_CURLY_BRACKET = "}";
   public static final String COMMA_SEPARATOR = ",";
+  public static final String EQUAL_SIGN = "=";
```

Comment:
I need it for the Export and Import to check where the file contains equal signs. Since I used only constants in my Import/Export classes and the only variable was this equal sign, I changed the equal sign to a constant.